repose-aggregator/components/filters/destination-router-filter/src/test/scala/org/openrepose/filters/destinationrouter/DestinationRouterFilterTest.scala

```diff
@@ -38,13 +38,19 @@ import org.springframework.mock.web.{MockFilterChain, MockFilterConfig, MockHttp
 @RunWith(classOf[JUnitRunner])
 class DestinationRouterFilterTest extends FunSpec with Matchers with BeforeAndAfterEach with MockitoSugar {
 
-  val configurationService = mock[ConfigurationService]
-  val meterByCategorySum = mock[MeterByCategorySum]
-  val metricsService = mock[MetricsService]
+  private val configurationService = mock[ConfigurationService]
+  private val metricsService = mock[MetricsService]
+  private val metricRegistry = mock[MetricRegistry]
+  private val meter = mock[Meter]
 
-  override def beforeEach() = {
+  override def beforeEach(): Unit = {
     reset(configurationService)
     reset(metricsService)
+    reset(metricRegistry)
+    reset(meter)
```

Comment:
I know that you're following the pattern this test was already using, but I super don't like having to call `reset` on mock objects. Then you have to rely on mutable state being in the right state across tests. I much prefer just creating new mocks for each test.

For more reasoning, I like this answer:
http://stackoverflow.com/a/11386867